apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx

```diff
@@ -3,11 +3,17 @@ import { displayName } from 'name';
 import { displayAddress } from 'address/display';
 import { Link } from 'react-router-dom';
 
-const displayProfileLink = (result: PatientSearchResult) => (
-    <Link to={`/patient-profile/${result.shortId}/summary`}>
-        {(result.legalName && displayName('fullLastFirst')(result.legalName)) || 'No Data'}
-    </Link>
-);
+const displayProfileLink = (result: PatientSearchResult) => {
+    return <Link to={`/patient-profile/${result.shortId}/summary`}>{result.shortId || 'No Data'}</Link>;
```

Comment:
nit: a patient will always have a `shortId` so the "No Data" should not be required.

suggestion (non-blocking): The `displayProfileLink` function could be changed to accept a `string` instead.  That way showing the link for the Patient ID column would become `displayProfileLink(result.shortId)` and the `displayProfileLegalName` function would call `displayName` and pass that to `displayProfileLink`.  This would remove the duplicate code for creating the link.